### PR TITLE
Patch Vanilla Factions Expanded - Tribals

### PR DESCRIPTION
**Pull request by N7Huntsman**
## Additions
- Patch for Vanilla Factions Expanded - Tribals

## Changes
- Update armor penetration values in Vanilla Weapons Expanded - Tribal that were using the old armor penetration node.

## Reasoning
- New tribal gear is balanced on par with existing tribalwear and simple weapons.

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [x] Playtested a colony (specify how long)

### CODE REVIEW

File: Patches/Vanilla Factions Expanded - Tribals/ThingDefs_WeaponsRanged.xml [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+						<ThingDef ParentName="BaseWeaponAndAmmoNeolithic" Name="SpikeBase_VFET" Abstract="True" />
+
+						<ThingDef ParentName="BasePilumProjectile">
+							<defName>Tribal_Spike_Thrown</defName>
+							<label>throwspike</label>
+							<projectile Class="CombatExtended.ProjectilePropertiesCE">
+								<damageAmountBase>14</damageAmountBase>
+								<speed>10</speed>
+								<armorPenetrationBlunt>6.44</armorPenetrationBlunt>
+								<armorPenetrationSharp>3.5</armorPenetrationSharp>
```
Comment (Safairette) [inline Patches/Vanilla Factions Expanded - Tribals/ThingDefs_WeaponsRanged.xml:27]: That seems rather... high for hand thrown wooden spikes.

**Review by Safairette — CHANGES_REQUESTED** [review]
A couple issues with the thrown spike, but otherwise seems to work fine.

File: Patches/Vanilla Factions Expanded - Tribals/ThingDefs_WeaponsRanged.xml (hunk 1 above)
Comment (N7Huntsman) [reply Patches/Vanilla Factions Expanded - Tribals/ThingDefs_WeaponsRanged.xml:27]: Good catch--I copy-pasted the patch from the VFE: Classical pilum, and apparently my commit with some balance tweaks didn't make it in.

File: Patches/Vanilla Factions Expanded - Tribals/ThingDefs_WeaponsRanged.xml [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+							<Bulk>1.5</Bulk>
+							<Mass>1</Mass>
+							<RangedWeapon_Cooldown>1.3</RangedWeapon_Cooldown>
+						</statBases>
+						<stackLimit>50</stackLimit>
+					</value>
+				</li>
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="VFET_Throwspikes"]/verbs</xpath>
```
[the remark this answers is not in the document]
Comment (N7Huntsman) [reply Patches/Vanilla Factions Expanded - Tribals/ThingDefs_WeaponsRanged.xml:63]: fixed

**Review by Safairette — APPROVED** [review]

**Pull request merged**